modules/biobank/php/pool.class.inc

```diff
@@ -0,0 +1,266 @@
+{
+    /**
+     * Persistent Instance variables.
+     *
+     * These variables are directly mapped to the fields of the database schema
+     * that describe a pool entity.
+     *
+     * @access private
+     */
+    private $id;          //int
+    private $label;       //string
+    private $specimenIds; //array
+    private $candidateId; //int
+    private $sessionId;   //int
+    private $centerId;    //int
+    private $date;        //string
+    private $time;        //string
+
+    /**
+     * Initiliazes a new instance of the Pool Class
+     */
+    function __construct()
+    {
+    }
+
+    /**
+     * Sets the pool's ID
+     *
+     * @param int $id the pool's ID
+     *
+     * @return void
+     */
+    public function setId(?int $id = null) : void
+    {
+        $this->id = $id;
+    }
+
+    /**
+     * Gets the pool's ID
+     *
+     * @return int
+     */
+    public function getId() : ?int
+    {
+        return $this->id;
+    }
+
+    /**
+     * Sets the label of the pool
+     *
+     * @param string $label the pool's label
+     *
+     * @return void
+     */
+    public function setLabel(string $label) : void
+    {
+        $this->label = $label;
+    }
+
+    /**
+     * Gets the label of the pool
+     *
+     * @return string
+     */
+    public function getLabel() : string
+    {
+        return $this->label;
+    }
+
+    /**
+     * Sets the Specimen IDs of the pool
+     *
+     * @param array $specimenIds Specimen IDs of the pool
+     *
+     * @return void
+     */
+    public function setSpecimenIds(array $specimenIds) : void
+    {
+        $this->specimenIds = $specimenIds;
+    }
+
+    /**
+     * Gets the Specimen IDs of the pool
+     *
+     * @return array
+     */
+    public function getSpecimenIds() : array
+    {
+        return $this->specimenIds;
+    }
+
+    /**
+     * Sets the Candidate ID of the pool
+     *
+     * @param int $candidateId Candidate ID of the pool
+     *
+     * @return void
+     */
+    public function setCandidateId(int $candidateId) : void
+    {
+        $this->candidateId = $candidateId;
+    }
+
+    /**
+     * Gets the Candidate ID of the pool
+     *
+     * @return int
+     */
+    public function getCandidateId() : int
+    {
+        return $this->candidateId;
+    }
+
+    /**
+     * Sets the Session ID of the pool
+     *
+     * @param int $sessionId Session ID of the pool
+     *
+     * @return void
+     */
+    public function setSessionId(int $sessionId) : void
+    {
+        $this->sessionId = $sessionId;
+    }
+
+    /**
+     * Gets the Session ID of the pool
+     *
+     * @return int
+     */
+    public function getSessionId() : int
+    {
+        return $this->sessionId;
+    }
+
+    /**
+     * Sets the Center ID of the pool
+     *
+     * @param array $centerId Center ID of the pool
+     *
+     * @return void
+     */
+    public function setCenterId(int $centerId) : void
```

Comment:
Maybe capitalize the "d" in setCenterId so it becomes setCenterID and similar to the get method below.

Reply:
Yeah, the upper case `ID` on line 185 is a mistake. I'll fix that!